src/buildercore/core.py

```diff
@@ -668,8 +668,8 @@ def hostname_struct(stackname):
         return struct
 
     # removes any non-alphanumeric or hyphen characters
-    subsubdomain = re.sub(r'[^\w\-]', '', instance_id)
-    hostname = subsubdomain + "--" + subdomain
+    instance_subdomain_fragment = re.sub(r'[^\w\-]', '', instance_id)
```

Comment:
ha `subsubdomain` - I can tell that was me, once upon a time

Reply:
also, it's no longer a sub-sub-domain anyway, we got rid of that ages ago. huh. code archaeology